strategy/api/resources/templates/class.cfm

```diff
@@ -688,6 +688,28 @@ Class
 		}
 		*/
 
+		/**
+		 * Improved version of Ben Forta's "ParagraphFormat2" method on cflib.
+		 * 
+		 * @cite https://cflib.org/udf/ParagraphFormat2
+		 * @param string      The string to format. (Required)
+		 * @return Returns a string. 
+		 * @author Ben Forta (ben@forta.com) 
+		 * @version 3, June 26, 2002 
+		 */
+		function formatLineBreaks( string val ){
+			// first make Windows style into Unix style
+			val = replace(val,chr(13)&chr(10),chr(10),"ALL");
```

Comment:
can you make sure you use the `arguments` scope.

Reply:
Nice catch! Yep, I can do that. That's what I get for using CFLib functions from 2002...